### PR TITLE
Improve RepoManagementController

### PR DESCRIPTION
**Pull request by JAmmermann-DLR**
This PR fixes the handling of the .properties file.

- in ServerRepoHelper:
  - registering a repository (project) now creates the .properties file
  - deleting also deletes the files
  - updating also saves the updated file
-  add test cases in ServerRepoHelper
- add test for removing a project in ServerRepositoryTest and minor fix in ServerRepository
- minor improvements in RepoManagementController and ProjectManagementResource

Closes https://github.com/virtualsatellite/VirtualSatellite4-Core/issues/556

### CODE REVIEW

**Comment by JAmmermann-DLR** [comment]
I noticed we still don't checkout/remove a repository at runtime when using the management REST API. I think have to add this to the ServerRepoHelper logic, do you agree? I would create a new ticket for this.

File: de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/repository/ServerRepoHelperTest.java [hunk 1]
```diff
@@ -38,6 +38,10 @@
 	private Path configsDir;
 	private Path svnPathRepoRemote;
 	private URI svnUriToRemoteRepoPath;
+	private static final String PROJECT_NAME = "testProject";
+	private static final String URI = "test.uri";
+	Path configFilePath;
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/repository/ServerRepoHelperTest.java:43]: private?

**Review by andriikovalov-dlr — CHANGES_REQUESTED** [review]
Looks good, just one minor point

**Review by SaMuellerDLR — APPROVED** [review]
Looking clean!

**Review by andriikovalov-dlr — APPROVED** [review]

**Pull request merged**